Source/interactivity_manager_impl.cpp

```diff
@@ -1167,6 +1187,10 @@ interactivity_manager_impl::on_socket_message_received(
     _In_ const string_t& message
 )
 {
+    // We always return all messages from the service to to the developer, even the ones the SDK will process them.
+    std::shared_ptr<interactive_message_event_args> args = std::shared_ptr<interactive_message_event_args>(new interactive_message_event_args(message));
+    queue_interactive_event_for_client(L"", std::error_code(0, std::generic_category()), interactive_event_type::unknown, args);

```

Comment:
If I'm reading this right... you're sending ALL messages, in raw json, out to the client? As well as sending some of the messages in object types? This seems off. If you want to send raw json for all messages, just do that, don't do both.

---
In reply to: [132343213](https://github.com/mixer/interactive-cpp/pull/25#discussion_r132343213) [](ancestors = 132343213)